### PR TITLE
Offer card

### PR DESCRIPTION
**Pull request by abarmina**

### CODE REVIEW

**Review by Zairbekov — APPROVED** [review]

File: src/app/shared/custom-components/offer-item/offer-item.component.scss [hunk 1]
```diff
@@ -1,135 +1,157 @@
 @import '/src/app/styles/variables.scss';
-
-:host::ng-deep .mat-card-header-text {
-  margin: 0.6em;
-}
+$small-card: 1500px;
```
Comment (tlazarchuk) [inline src/app/shared/custom-components/offer-item/offer-item.component.scss:4]: You might leave a comment on what is it or even better put this into our special file with CSS variables

File: src/app/shared/custom-components/offer-item/offer-item.component.scss [hunk 2]
```diff
@@ -1,135 +1,157 @@
...
-    }
+  &__statistics {
+    display: flex;
+    justify-content: space-around;
+    color: $background-icon;
 
-    &__discount {
-      font-size: 16px;
+    @media screen and (max-width: $small-card) {
+      order: -1;
```
Comment (tlazarchuk) [inline src/app/shared/custom-components/offer-item/offer-item.component.scss:89]: You should leave a comment. I don't understand what it has to do

**Review by tlazarchuk — APPROVED** [review]

File: src/app/shared/custom-components/offer-item/offer-item.component.scss (hunk 1 above)
Comment (abarmina) [reply src/app/shared/custom-components/offer-item/offer-item.component.scss:4]: this is the width for the card only, I doubt that we will need this value in other places, so I defined it only in this component. I can leave a comment

File: src/app/shared/custom-components/offer-item/offer-item.component.scss (hunk 2 above)
Comment (abarmina) [reply src/app/shared/custom-components/offer-item/offer-item.component.scss:89]: ok

**Pull request merged**